include/faker-cxx/Sport.h

```diff
@@ -2,11 +2,8 @@
 
 #include <string_view>
 
-namespace faker
+namespace faker::Sport
```

Comment:
Please use lowercase for namespaces as in other modules

Reply:
modified namespaces to lowecase